tests/python_functional/src/conftest.py

```diff
@@ -39,7 +39,7 @@ def fake_testcase_parameters(request, tmpdir):
     tc_parameters.WORKING_DIR = tmpdir.join("workingdir")
     request.config.option.installdir = tmpdir.join("installdir")
     request.config.option.reports = tmpdir.join("reports")
-    request.config.option.runwithvalgrind = False
+    request.config.option.runundertool = False
```

Comment:
I would change the default to `""`, as we are expecting strings here.

Reply:
Thanks for the review note. I have fixed this.